### ISSUE TITLE
Suggest new PIPE

### ISSUE THREAD

**Issue by viniciosyals** [danrevah/ngx-pipes#102]
```
[X] feature request
```
A TimeAgo PIPE (like this https://github.com/AndrewPoyntz/time-ago-pipe)

|Time Range|Output|
|---|---|
|0 - 45 seconds             | a few seconds ago      |
|45 - 90 seconds            | a minute ago           |
|90 seconds - 45 minutes    | X minutes ago          |
|45 - 90 minutes            | an hour ago            |
|90 minutes - 22 hours      | X hours ago            |
|22 - 36 hours              | a day ago              |
|36 hours - 25 days         | X days ago             |
|25 - 45 days               | a month ago            |
|45 - 345 days              | X months ago           |
|345 - 545 days (1.5 years) | a year ago             |
|546 days+                  | X years ago            |

```
date = '2018-02-08T00:00:00'
```
```
{{date | timeAgo}} // 2 days ago
```

With supports internationalization

**Comment by danrevah** [comment]
Seems to me like it can fit to `ngx-pipes`.

Would you like to make a PR for this?

Or, does anyone else wants?

**Comment by matthew-valenti** [comment]
The popular momentjs.com plugin supports something similar with moment-duration-format: https://github.com/jsmreese/moment-duration-format

**Comment by cpilson** [comment]
I will admit my confusion as well. If someone is interested in dates, it seems to me they’d have already pulled in `moment` or `moment-timezone`. And as the other commenter indicates, they can format times like this.

**Comment by christopheelkhoury** [comment]
Even though the guys above do have a point, I still think it'd be a nice added value as a pipe. If anyone hasn't adopted the idea and started working on it, I'll take it.

**Comment by danrevah** [comment]
@christopheelkhoury Go ahead :)

**Comment by christopheelkhoury** [comment]
On it.
Mind if you add me as an assignee?

**Comment by JinnaBalu** [comment]
25 - 45 days -  how can this be one month ago? some thing we may need to correct.

**Comment by danrevah** [comment]
Sorry for the delay guys.. someone didn't finish his pull-request so I've just fixed all the issues and published it in version `v2.4.5`.

Enjoy!
